cmd/stacker/main.go

```diff
@@ -225,13 +229,30 @@ func main() {
 
 		config.EmbeddedFS = embeddedFS
 
-		if config.StackerDir == "" || ctx.IsSet("stacker-dir") {
+		if ctx.IsSet("work-dir") {
+			config.WorkDir = ctx.String("work-dir")
+		}
+		if ctx.IsSet("stacker-dir") {
+			config.StackerDir = ctx.String("stacker-dir")
+		} else if ctx.IsSet("work-dir") {
```

Comment:
These three hunks don't make sense to me.  Does 'ctx.IsSet(X)' mean something other than what I'd think?

You're saying
```
if ctx.IsSet("stacker-dir) {
   config.StackerDir = ctx.String("stacker-dir")
} else if ctx.IsSet("work-dir") {
  config.StackerDir = path.Join(ctx.String("work-dir"), ctx.String("stacker-dir")) // but ctx.IsSet("stacker-dir") is false!
```

Reply:
https://github.com/project-stacker/stacker/pull/378#issuecomment-1407024467

^ trying to achieve this

Reply:
Oh, I see - IsSet() threw me off because it's smarter than I thought.  ctx.String("stacker-dir") will return the default value while ctx.IsSet("stacker-dir") returns "false.

Additionally, 'work-dir' left me confused last night as it makes me think of the overlay 'work-dir'.  But I don't have a better suggestion.